portable/GCC/AVR32_UC3/exception.S

```diff
@@ -9,8 +9,8 @@
  * - Supported devices:  All AVR32UC devices with an INTC module can be used.
  * - AppNote:
  *
- * \author               Atmel Corporation: http://www.atmel.com \n
- *                       Support and FAQ: http://support.atmel.no/
+ * \author               Atmel Corporation: https://www.microchip.com \n

```

Comment:
Please change to  "Atmel Corporation (Now Microchip)" - then it will make sense next to the microchip.com URL.  (so add the "Now Microchip" part).  Same comment for each occurrence below.

Reply:
done